### DISCUSSION TITLE
Scheduled Tasks not being created ?

### DISCUSSION THREAD

**Discussion by ja-law** [Romanitho/Winget-AutoUpdate#504] [Q&A]
Anyone else experienced the scheduled tasks not being created now ?

**Comment by KnifMelti** [comment]
No, not really. Looking in the right place?

![image](https://github.com/Romanitho/Winget-AutoUpdate/assets/102996177/a40b50e3-ebf3-4de5-ad2a-b34f4e531aab)

**Reply by ja-law** [reply]
Been working for the last couple of months and deployed to many machines, now when I deploy it doesn't create any of the scheduled tasks when I run Winget-AutoUpdate-Install.ps1

**Comment by ja-law** [comment]
Tried with latest version and no scheduled tasks created

Winget-AutoUpdate-Install -DoNotUpdate -UpdatesAtLogon -UpdatesInterval Never -InstallUserContext -DesktopShortcut -ListPath https://...............................................

**Comment by ja-law** [comment]
Didn't look in the WAU folder in Scheduled Tasks

**Reply by Romanitho** [reply]
So, it's ok?

**Reply by ja-law** [reply]
Yes, the scheduled tasks used to be in the root, but now in the WAU folder.

**Reply by Romanitho** [reply]
Yes, we needed to tidy things up :)

**Reply by ja-law** [reply]
Ok, didn't see it mentioned in release notes.

**Reply by Romanitho** [reply]
Sorry about that, it was part of the GPO rework:
https://github.com/Romanitho/Winget-AutoUpdate/pull/410

**Reply by ja-law** [reply]
no probs
